processor/src/fast/mod.rs

```diff
@@ -302,29 +302,25 @@ impl FastProcessor {
     // -------------------------------------------------------------------------------------------
 
     /// Executes the given program and returns the stack outputs.
-    pub fn execute(
+    pub async fn execute(
         mut self,
         program: &Program,
         host: &mut impl Host,
```

Comment:
Shouldn't this take `AsyncHost` as a parameter?

Reply:
Oops - fixed